functionalities-guide/cockpit/index.rst

```diff
@@ -597,7 +597,9 @@ This option allows the user to manage all cockpit general settings that we are g
 
     General configuration window.
 
-Editing the fields of the first tab you can add or change the name and/or the description of your cockpit; moreover here you can choose the sheet color and decide to enable the menu when the document runs in display mode. The second tab (Figure below allows to configure some style options of the cockpit, like borders, shadows, titles and background color.
+Editing the fields of the first tab you can add or change the name and/or the description of your cockpit; moreover here you can choose the sheet color or a background image and its size. In particular, in order to add a background image for the sheets, firstly you have to add the image to the catalogue of the image widget and then copy the link of the image. It is also possible to decide to enable the menu and the widgets functionalities when the document runs in display mode or to disable the screenshot functionality for every widgets. 
+
+The second tab (Figure below allows to configure some style options of the cockpit, like borders, shadows, titles and background color.
```

Comment:
The second tab (Figure below) allows configuring some style options of the cockpit, like borders, shadows, titles and background color.